Sources/XcbeautifyLib/Parser.swift

```diff
@@ -67,7 +69,7 @@ public class Parser {
         innerParser(.willNotBeCodeSigned, outputType: .warning),
         innerParser(.clangError, outputType: .error),
         innerParser(.checkDependenciesErrors, outputType: .error),
-        innerParser(.provisioningProfileRequired, outputType: .warning),
+        innerParser(.provisioningProfileRequired, outputType: .error),
```

Comment:
Looks like this wasn't caught in previous releases, but - looking at the existing implementation and tests - I think this and the one below should be an error.

Reply:
Is this change worth moving to a separate PR, so as not to couple the new functionality of this PR with this (correct) change?

Reply:
Good call! I created this PR: https://github.com/tuist/xcbeautify/pull/123.